sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/RntbdTransportClient.java

```diff
@@ -566,7 +599,6 @@ public String toString() {
          *   "maxRequestsPerChannel": 30,
          *   "maxConcurrentRequestsPerEndpointOverride": 500,
          *   "receiveHangDetectionTime": "PT1M5S",
-         *   "requestExpiryInterval": "PT5S",
```

Comment:
I think we should remove these, since DirectConnectionConfig already provides the APIs now. This was before the public APIs and was only used through JVM options. 
Or if we want to keep these configurations, we should update them to the correct values - like `maxChannelsPerEndpoint`, `maxRequestsPerChannel`, `idleEndpointTimeout`, etc.

Reply:
hmm, I think this comments are more about giving different ways how you can using the JVM options and setup the values. I feel it has been very useful when doing tests. 

So for now, I left the comments to be the way it is today, but update the values to be consistent with the default values.